src/pages/EndpointManager.vue

```diff
@@ -103,12 +103,17 @@ export default {
     },
     uiThemeCategory: {
       get() {
-        let zclProps = this.$store.state.zap.selectedZapConfig?.zclProperties
-        // Picking the first category in the case of multi-protocol(zigbee/matter)
-        if (Array.isArray(zclProps) && zclProps.length > 0) {
-          return zclProps[0].category
+        if (this.$store.state.zap.isMultiConfig) {
+          return 'multiprotocol'
         } else {
-          return this.$store.state.zap.selectedZapConfig?.zclProperties.category
+          let zclProps = this.$store.state.zap.selectedZapConfig?.zclProperties
+          // Picking the first category in the case of multi-protocol(zigbee/matter)
+          if (Array.isArray(zclProps) && zclProps.length > 0) {
+            return zclProps[0].category
+          } else {
```

Comment:
Isn't `if (Array.isArray(zclProps) && zclProps.length > 0) {
             return zclProps[0].category
           }` and `if (this.$store.state.zap.isMultiConfig) {
           return 'multiprotocol'
         } ` the same?

Reply:
Should this all be something like

`if (this.$store.state.zap.isMultiConfig) {
           return 'multiprotocol'
         } else {
           return this.$store.state.zap.selectedZapConfig?.zclProperties.category
          }`

Reply:
i think no, because if user choose two zigbee or two matter categories (if it is possible) they are not multiprotocol but zclProperties is an arrray.  if it is not possible, you are right.